src/app/features/vendor/models/vendor.model.ts

```diff
@@ -0,0 +1,12 @@
+export interface Vendor {
+  id: string;
+  name: string;
+  state?:string;
+  country:string;
+  markets: string[];
```

Comment:
File is not formatted. (space is missing after colon)

Reply:
format corrected